<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="2598" started-at="2018-10-27T12:01:38Z" finished-at="2018-10-27T12:01:41Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="2598" started-at="2018-10-27T12:01:38Z" finished-at="2018-10-27T12:01:41Z">
      <class name="in.licious.dataplanning_API.DashboardWastageAPI">
        <test-method status="FAIL" signature="dashboardwastagetest()[pri:0, instance:in.licious.dataplanning_API.DashboardWastageAPI@4883b407]" name="dashboardwastagetest" duration-ms="2574" started-at="2018-10-27T17:31:38Z" finished-at="2018-10-27T17:31:41Z">
          <exception class="java.net.MalformedURLException">
            <message>
              <![CDATA[no protocol: 52.66.9.219:9200/_sql?sql=select sum(dispatched_quantity) from operationdataplan where to='11' and source='1' and createdat='2018-10-22' and sku='pr_57234a4f6f77b']]>
            </message>
            <full-stacktrace>
              <![CDATA[java.net.MalformedURLException: no protocol: 52.66.9.219:9200/_sql?sql=select sum(dispatched_quantity) from operationdataplan where to='11' and source='1' and createdat='2018-10-22' and sku='pr_57234a4f6f77b'
	at java.net.URL.<init>(URL.java:593)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at in.licious.dataplanning.ElasticSearch.readJsonFromUrl(ElasticSearch.java:28)
	at in.licious.dataplanning_API.DashboardWastageAPI.dashboardwastagetest(DashboardWastageAPI.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.net.MalformedURLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashboardwastagetest -->
      </class> <!-- in.licious.dataplanning_API.DashboardWastageAPI -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
